ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/DLModelExecute.java

```diff
@@ -52,7 +52,7 @@ public abstract class DLModelExecute implements MLExecutable {
     protected Device[] devices;
     protected AtomicInteger nextDevice = new AtomicInteger(0);
 
-    public abstract void execute(Input input, String tenantId, ActionListener<Output> listener);
+    public abstract void execute(Input input, ActionListener<Output> listener);
```

Comment:
Is tenantId removed since it is part of `input`?

Reply:
yes